src/transport/SessionManager.cpp

```diff
@@ -478,7 +478,7 @@ CHIP_ERROR SessionManager::SendPreparedMessage(const SessionHandle & sessionHand
     {
         CHIP_ERROR err = mTransportMgr->SendMessage(*destination, std::move(msgBuf));
 #if CHIP_ERROR_LOGGING
-        if (err != CHIP_NO_ERROR)
```

Comment:
The existing check works fine, you must have been running a build from the brief time between when the logging was introduced and when the `err != CHIP_NO_ERROR` check was added.